frontend/src/config/usersAPI.ts

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+import type { User } from './user.types';
+
+const api = axios.create({ baseURL: 'http://localhost:4000/' });
+
+const usersData = (res: { data: User[] }) => res.data;
```

Comment:
Functions do some action, so the name of the fn should always starts from action, like what exactly this fn does
```suggestion
const extractUsers = (res: { data: User[] }) => res.data;
```